ci/tasks/bump-yq-packages/task.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+NEED_COMMIT=false
+
+set -euo pipefail
+
+echo "${PRIVATE_YML}" > postgres-release/config/private.yml
+
+pushd postgres-release
+  CURRENT_BLOBS=$(bosh blobs)
+  CURRENT_VERSION=$(curl -s -L https://api.github.com/repos/mikefarah/yq/releases/latest | grep "tag_name" | sed s/\"tag_name\":\//g | sed s/\"//g | sed s/\,//g | sed s/v//g | xargs)
```

Comment:
Although it's not documented on the resource page, the http-resource we use in concourse here will actually drop a `version` file when fetching the resource.

So `../yq-src/version` should contain the version number of the yq that was just fetched rather than having to fetch it again here.

Reply:
Done